src/dbos-runtime/runtime.ts

```diff
@@ -65,17 +63,9 @@ export class DBOSRuntime {
   startServer() {
     // CLI takes precedence over config file, which takes precedence over default config.
 
-    const server: DBOSHttpServer = new DBOSHttpServer(this.dbosExec)
+    const server = new DBOSHttpServer(this.dbosExec)
 
-    this.server = server.listen(this.runtimeConfig.port);
+    server.listen(this.runtimeConfig.port);
     this.dbosExec.logRegisteredHTTPUrls();
   }
-
-  /**
-   * Shut down the HTTP server and destroy workflow executor.
-   */
-  async destroy() {
```

Comment:
This is a slight conflict with my PR. I added a piece that actually calls this and I use it to flush the errors:
https://github.com/dbos-inc/dbos-ts/pull/259/files

Reply:
Removed it because it wasn't being used, just saw your PR and added it back